src/interfaces/filing-interfaces/filingHeader-interface.ts

```diff
@@ -18,4 +18,6 @@ export interface FilingHeaderIF {
   datNumber?: string
   waiveFees?: boolean
   priority?: boolean
+
+  [propName: string]: any; // excess properties
```

Comment:
This is allow the addition of arbitrary fields at runtime (ie, isTransactionalFolioNumber),

Reply:
Is this a typed interface that accepts any type?